### DISCUSSION TITLE
[URGENT HELP] If you delete a directory from the "Workspace" browser, how can you restore it?

### DISCUSSION THREAD

**Discussion by ceruleo-dev** [flxzt/rnote#1135] [Q&A]
I was running Rnote Flatpak in Kubuntu 22.04 and I accidentally deleted the "/home/Documents" directory.
How can I restore it? I didn't found it in the Trash bin. It was permanently deleted?? I have all my data in there and no backup.
Does Flatpak sends the deleted directory to a particular place?
Please help me, I'm desperate and I can't find all my documents. Thanks in advance.

**Comment by anesthetice** [comment]
Using "Move to Trash" on a folder works for me, rnote uses [trash_async](https://docs.gtk.org/gio/method.File.trash_async.html) under the hood, do you have a trash management package installed? especially glib2?
https://wiki.archlinux.org/title/Trash_management

Install glib2, and try running the following in your terminal
`gio trash --list | grep ".rnote"`

**Comment by flxzt** [comment]
This probably is resolved, they opened issue #1136 shortly after

**Reply by anesthetice** [reply]
(I realized it was closed after responding ): )
